src/components/ebay-table/index.marko

```diff
@@ -23,6 +28,15 @@ $ const {
     <table>
         <thead>
             <tr>
+                <if(mode === "selection")>
+                    <th class="table__cell">
+                        <ebay-tri-state-checkbox
+                            aria-label=(a11ySelectAllText ?? "Select all rows")
+                            checked=component.getSelectionAttrTriState(input)
```

Comment:
This should probably be a state variable which is set on input or when the checked state of the other checkboxes is set.